frontend/src/e2e-test-common/dev-api/fixtures.ts

```diff
@@ -473,76 +473,83 @@ const applicationForm = (
   otherGuardianAgreementStatus: OtherGuardianAgreementStatus = null,
   preferredUnits: string[] = [daycareFixture.id],
   connectedDaycare = false
-): ApplicationForm => ({
-  type,
-  additionalDetails: {
-    allergyType: '',
-    dietType: '',
-    otherInfo: ''
-  },
-  apply: {
-    preferredUnits: preferredUnits,
-    siblingBasis: false
-  },
-  careDetails: {
-    assistanceNeeded: false
-  },
-  child: {
-    firstName: child.firstName,
-    lastName: child.lastName,
-    socialSecurityNumber: child.ssn,
-    address: {
-      street: child.streetAddress,
-      postalCode: child.postalCode,
-      city: child.postOffice,
-      editable: false
+): ApplicationForm => {
+  let startDate = new Date()
+  if (type === 'PRESCHOOL' && [5, 6, 7, 8].includes(getMonth(startDate) + 1)) {
+    // Try to make sure there's an active preschool term for the preferred start date
+    startDate = setMonth(startDate, 9)
```

Comment:
`setMonth(.., 9)` results in October. Not that it matters, but the commit message is conflicting.

Reply:
Le fu-